packages/web/pages/settings/integrations.tsx

```diff
@@ -146,18 +146,32 @@ export default function Integrations(): JSX.Element {
         },
       },
       {
-        icon: '/static/icons/readwise.svg',
-        title: 'Readwise',
+        icon: '/static/icons/obsidian.png',
+        title: 'Obsidian',
         subText:
-          'Readwise makes it easy to revisit and learn from your ebook & article highlights. Use our Readwise integration to sync your highlights from Omnivore to Readwise.',
+          'Obsidian is a powerful and extensible knowledge base that works on top of your local folder of plain text files. Use the Omnivore Obsidian plugin to sync articles, highlights, and notes to Obsidian.',
         button: {
-          text: readwiseConnected ? 'Remove' : 'Connect to Readwise',
+          text: `Install Logseq Plugin`,
```

Comment:
A typo here. Should be `Install Obsidian Plugin`. Typical C&P 😄

Reply:
eeeekkkkkk!